tests/Tests/Unit/CredentialsWrapperTest.php

```diff
@@ -47,20 +47,26 @@
 class CredentialsWrapperTest extends TestCase
 {
     /**
-     * @dataProvider buildData
+     * @dataProvider buildDataWithoutExplicitKeyFile
      */
-    public function testBuild($args, $expectedCredentialsWrapper)
+    public function testBuildWithoutExplicitKeyFile($args, $expectedCredentialsWrapper)
     {
+        putenv('GOOGLE_APPLICATION_CREDENTIALS=' . __DIR__ . '/testdata/json-key-file.json');
         $actualCredentialsWrapper = CredentialsWrapper::build($args);
         $this->assertEquals($expectedCredentialsWrapper, $actualCredentialsWrapper);
+        putenv('GOOGLE_APPLICATION_CREDENTIALS');
```

Comment:
Would it be better to run this in a separate process to make sure the env modifications don't effect other tests?